### DISCUSSION TITLE
[FEATURE REQUEST]: Better .Net Spark project developer experience

### DISCUSSION THREAD

**Discussion by pherbel** [dotnet/spark#584] [Ideas]
**Is your feature request related to a problem? Please describe.**
The main problem is the project's start experience and debugging.
I would like to start, test and debug my spark up from the visual studio.

**Describe the solution you'd like**
- .Net Spark project template
- Start from visual studio with debugging. Perhaps it can be achieved with separated Spark MsBuild SDK.
`<Project Sdk="Microsoft.NET.Sdk.Spark">`

**Comment by rapoth** [comment]
@pherbel Thank you for the suggestion! This has been on our radar but we did not get time to go behind this. Would you be willing to collaborate and contribute towards this feature? We would be happy to have a meeting, if that helps. Thank you!
